### ISSUE TITLE
Supporting for detection of `link[rel=shortcut]`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#715]
>For historical reasons, the `icon` keyword may be preceded by the keyword "`shortcut`". If the "`shortcut`" keyword is present, the `rel` attribute's entire value must be an ASCII case-insensitive match for the string "`shortcut icon`" (with a single U+0020 SPACE character between the tokens and no other ASCII whitespace).
https://html.spec.whatwg.org/multipage/links.html#linkTypes

# Examples

## Invalid

```html
<!-- Invalid (OK, but should not be used) -->
<link rel="shortcut icon" href="favicon.ico">
```

```html
<!-- Invalid -->
<link rel="shortcut" href="favicon.ico">
```

## Valid

```html
<!-- Valid -->
<link rel="icon" href="favicon.ico">
```
